frontend/src/pages/Risk/Risk.tsx

```diff
@@ -550,7 +550,10 @@ const Risk: React.FC = (props) => {
   const delay = (ms: number) => new Promise((res) => setTimeout(res, ms));
 
   const generatePDF = async () => {
+    const dateTimeNow = new Date();
+    const localDate = new Date(dateTimeNow);
     setIsLoading(true);
+    await delay(650);
```

Comment:
Why the delay?

Reply:
That is what solves the rendering of the charts in the PDF.  I dropped it as low as I could.